### ISSUE TITLE
Attach to camera

### ISSUE THREAD

**Issue by archenroot** [Dene33/video_to_bvh#1]
Nice stuff. How hard would it be to attach to real camera stream, lets say either via:
1. /dev/video0
or
2. via rtms stream from IP camera
?

**Comment by Dene33** [comment]
There are two main problems here:
1. This approach is based on complex computations - a lot of GPU memory is needed to be able to run in real time. However you can run it on a very low FPS (1 fps or something like that).
2. Theoretically, you'll be okay (with low fps) with generating .csv files on the fly. But my script of creating .bvh files takes .csv from the whole video i.e. fully processed video. So another way of showing skeleton must be implemented.

So, basically you'll need to reimplement reading of generated .csv files. Also, I think if you want real time camera stream, there are may be other more suitable implementations of pose estimation on GitHub.
